HW9/AntonKoluh/guess_game_task1.py

```
@@ -0,0 +1,158 @@
+        for event in pygame.event.get():
+            mouse_pos = pygame.mouse.get_pos()
+            if event.type == pygame.QUIT:
+                running = False
+            if event.type == pygame.KEYDOWN:
+                if event.key == pygame.K_BACKSPACE:
+                    input_answer.txt = input_answer.txt[:-1]
+                elif event.key == pygame.K_RETURN:
+                    check_answer(input_answer.txt)
+                    pygame.time.wait(100)
+                elif event.key != pygame.K_RETURN:
+                    if event.unicode.isdigit():
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)